scraper.py

```diff
@@ -132,12 +155,28 @@ def clean_data(cell):
 
     doc = parse_source(content, encoding)
     listings = extract_data_listings(doc)
-    for listing in listings:
+    for listing in listings[:count]:
         metadata = extract_restuarant_metadata(listing)
         restuarant_data = extract_score_data(listing)
         restuarant_data.update(metadata)
-        print restuarant_data
+        # geo_data = get_geojson(restuarant_data)
+        # restuarant_data.update(geo_data)
+        yield restuarant_data
         print
 
     print len(listings)
-    # print listings[0].prettify()
+
+
+if __name__ == '__main__':
+    try:
+        count = int(sys.argv[2])
+    except IndexError:
+        count = 10
+    all_results = {'type': 'FeatureCollection', 'features': []}
+    for result in generate_results(len(sys.argv) == 1, count):
```

Comment:
There's a dependency between the count and 'new' in this script.  How does one specify a count other than the default while at the same time fetching new data from the live website?